openai-connector-demo/src_hd/com/openai/connector/demo/AIMailGenerator/AIMailGenerator.xhtml

```diff
@@ -0,0 +1,204 @@
+						</p:inputText>
+						<p:message for="most-important-skill" />
+					</div>
+
+					<!-- Row -->
+					<div class="p-field p-col-12 p-md-2 p-text-left p-text-md-right">
+						<p:outputLabel value="Interviewer name" for="interviewer-name">
+							<span class="ui-outputlabel-rfi">*</span>
+						</p:outputLabel>
+					</div>
+					<div class="p-field p-col-12 p-md-4">
+						<p:inputText id="interviewer-name"
+							value="#{data.candidateInformation.interviewerName}">
+							<f:validator validatorId="commonValidator" />
+						</p:inputText>
+						<p:message for="interviewer-name" />
+					</div>
+
+					<div class="p-field p-col-12 p-md-2 p-text-left p-text-md-right">
+						<p:outputLabel value="Interview date" for="interview-date">
+							<span class="ui-outputlabel-rfi">*</span>
+						</p:outputLabel>
+					</div>
+					<div class="p-field p-col-12 p-md-4">
+						<p:datePicker id="interview-date"
+							value="#{data.candidateInformation.interviewDate}"
+							showIcon="true" pattern="dd/MM/yyyy"
+							converterMessage="#{ivy.cms.co('/Labels/WrongDateFormat')}"
+							mask="true">
+							<f:validator validatorId="commonValidator" />
+						</p:datePicker>
+						<p:message for="interview-date" />
+					</div>
+
+					<!-- Row -->
+					<div class="p-field p-col-12 p-md-2 p-text-left p-text-md-right">
+						<p:outputLabel value="Interviewer position"
+							for="interviewer-job-position">
+							<span class="ui-outputlabel-rfi">*</span>
+						</p:outputLabel>
+					</div>
+					<div class="p-field p-col-12 p-md-4">
+						<p:selectOneMenu id="interviewer-job-position"
+							value="#{data.candidateInformation.interviewerPosition}">
+							<f:selectItem itemLabel="Select Job Position" itemValue="" />
+							<f:selectItem itemLabel="Software Developer"
+								itemValue="Software Developer" />
+							<f:selectItem itemLabel="Database Administrator"
+								itemValue="Database Administrator" />
+							<f:selectItem itemLabel="Network Engineer"
+								itemValue="Network Engineer" />
+							<f:selectItem itemLabel="System Analyst"
+								itemValue="System Analyst" />
+							<f:selectItem itemLabel="IT Project Manager"
+								itemValue="IT Project Manager" />
+							<f:validator validatorId="commonValidator" />
+						</p:selectOneMenu>
+						<p:message for="interviewer-job-position" />
+					</div>
+
+					<!-- Row -->
+					<div class="p-field p-col-12 p-md-2 p-text-left p-text-md-right">
+						<p:outputLabel value="Company" for="company">
+							<span class="ui-outputlabel-rfi">*</span>
+						</p:outputLabel>
+					</div>
+					<div class="p-field p-col-12 p-md-4">
+						<p:inputText id="company"
+							value="#{data.candidateInformation.company}">
+							<f:validator validatorId="commonValidator" />
+						</p:inputText>
+						<p:message for="company" />
+					</div>
+
+					<!-- Row -->
+					<div class="p-field p-col-12 p-md-2 p-text-left p-text-md-right">
```

Comment:
This row is not consistent with the other rows (its height is taller than the others).